test/test_einstein_kernels.py

```diff
@@ -34,7 +38,7 @@
     T(RandomFeatureKernel,
       lambda d: {},
       lambda x: x,
-      {'norm': 12.190277}),
+      {'norm': 15.486456}),
```

Comment:
@OlaRonning Because the random pattern is changed (using jax.random instead of numpy.random), this test failed. So I changed the number here. I think testing against a random seed is not a good way but have no other ideas.

Reply:
LGTM